frontend/components/atoms/Icons/index.tsx

```diff
@@ -104,4 +113,16 @@ const Icons = ({ name, size = '20', additionalClass = '' }: IconsProps): JSX.Ele
     }
 }
 
-export default Icons
+export const Icons = {
```

Comment:
What I think of `Old` as variable name is not in use.
You can name the new Icons as `CustomIcons` and rename back the OldIcons to `Icons`.